controllers/mondooauditconfig_controller.go

```diff
@@ -56,6 +64,11 @@ var (
 
 	// Defined as a var so we can easily mock this in tests.
 	createContainerImageResolver = mondoo.NewContainerImageResolver
+
+	// so we can mock out the mondoo client for testing
+	MondooClientBuilder = func(opts mondooclient.ClientOptions) mondooclient.Client {
```

Comment:
```suggestion
	MondooClientBuilder = mondooclient.NewClient
```

I think this is equivalent but then shorter